app/src/main/java/com/andela/omokolataiwo/levelup/models/GithubProfile.java

```diff
@@ -1,32 +1,103 @@
 package com.andela.omokolataiwo.levelup.models;
 
+import android.os.Parcel;
+import android.os.Parcelable;
+
 import com.google.gson.annotations.SerializedName;
 
-public class GithubProfile {
+/**
+ * A developer github profile.
+ */
+public class GithubProfile implements Parcelable {
```

Comment:
@omokolataiwo 
I can see you implemented `Parcelable` here. just make sure you do not do this 👇 
`outState.putParcelableArrayList(PARCEL_KEY, (ArrayList<? extends Parcelable>) githubProfilesParcel);`
in the `onSaveInstanceState` method of you `MainActivity` class

Reply:
I have to cast the field to ArrayList collection because putParcelableArrayList interface requires ArrayList of objects that extends Parcelable and I have used List collection of githubProfiles objects.

Reply:
@omokolataiwo 
I think i get you.... If you have used `ArrayList` type for all your model variable instances, that would have been possible.... But since you used `List` type instead, doing what did there is ok by me.
Great job.... Do merge immediately